pkg/plugins/mellanox/mellanox_plugin.go

```diff
@@ -111,6 +111,11 @@ func (p *MellanoxPlugin) OnNodeStateChange(old, new *sriovnetworkv1.SriovNetwork
 		}
 		processedNics[pciPrefix] = true
 
+		if utils.IsKernelLockdownMode() {
```

Comment:
Can we combine utils.IsKernelLockdownMode in line 114 and line 147 into one? and lift it to the beginning of OnNodeStateChange ? Since in lock down mode, there is no need to run the plugin.